docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -21,26 +21,26 @@ NOTE: If you have deployed the {WebName} by pasting in a JSON template, no image
 [id='examine_logs_{context}']
 == Examining and collecting {WebName} logs
 
-Each Pod is configured to provide detailed logging that assists with narrowing down the precise cause of an issue. The following steps discuss viewing and obtaining these logs.
+Each pod is configured to provide detailed logging that assists with narrowing down the precise cause of an issue. You can use the OpenShift Container Platform console or the CLI to download pod logs.
 
-.OpenShift console
+.OpenShift Container Platform console
 
-You can download a Pod's log with the OpenShift Container Platform console.
+You can download a pod log with the OpenShift Container Platform console.
 
 .Procedure
 
-. Open the OpenShift console and navigate to *Applications* -> *Pods*.
-. Click the name of the Pod that you wish to examine. To examine the current {WebName} Pod, select the `mta-web-console-<POD_NAME>` that is in a `Running` state.
+. Open the OpenShift Container Platform console and navigate to *Applications* -> *Pods*.
+. Click the name of the pod that you wish to examine. To examine the current `{WebName}` pod, select the `mta-web-console-<POD_NAME>` that is in a `Running` state.
```

Comment:
Can you combine this step with the next? `Click a pod and then click *Logs*.`

You don't need the second sentence. We can assume that users know that a "current" pod is in a Running state.